wiki/Tournaments/SS/2022/es.md

```diff
@@ -0,0 +1,631 @@
+---
+tags:
+  - SS
+  - SS2022
+  - SS 2022
+---
+
+# Spanish Showdown 2022
+
+![Logo del SS 2022](img/logo.jpg)
+
+El **Spanish Showdown 2022** (***SS 2022***) fue un torneo español de doble eliminación 1vs1 de osu! organizado por ::{ flag=ES }:: [E R N E S T O](https://osu.ppy.sh/users/2127413), ::{ flag=ES }:: [M A N O L O](https://osu.ppy.sh/users/12296128), y ::{ flag=ES }:: [Margo](https://osu.ppy.sh/users/6995685). El torneo iba dedicado a los mejores jugadores de osu! españoles, sin límite en su rango. Ha sido la segunda edición del Spanish Showdown.
```

Comment:
> El torneo iba dedicado a los mejores jugadores de osu! españoles, sin límite en su rango. Ha sido la segunda edición del Spanish Showdown.

Is this supposed to be in past tense?

Reply:
Correct, tournament already ended.
inb4 another wave of suggestions.